content/en/blog/2025/sampling-milestones.md

```diff
@@ -0,0 +1,240 @@
+---
+title: OpenTelemetry Sampling update
+linkTitle: OpenTelemetry Sampling update
+date: 2025-10-01
+author: >-
+  [Joshua MacDonald](https://github.com/jmacd) (Microsoft)
```

Comment:
@PeterF778
@kentquirk
@yuanyuanzhao3
@oertl 

I would like to list all of our names as co-authors! With your permission.